src/mc_rbdyn/configuration_io.cpp

```diff
@@ -162,8 +186,10 @@ mc_rtc::Configuration ConfigurationLoader<mc_rbdyn::Collision>::save(const mc_rb
   config.add("iDist", c.iDist);
   config.add("sDist", c.sDist);
   config.add("damping", c.damping);
-  config.add("r1Joints", c.r1Joints);
-  config.add("r2Joints", c.r2Joints);
+  if(c.r1Joints) { config.add("r1Joints", *c.r1Joints); }
+  if(c.r2Joints) { config.add("r2Joints", *c.r2Joints); }
+  config.add("r1JointsInactive", c.r1JointsInactive);
+  config.add("r2JointsInactive", c.r2JointsInactive);
```

Comment:
I think this can be omitted in the output object if either `r1Joints` and/or `r2Joints` are not provided.